app/common/locales/en.json

```diff
@@ -760,6 +760,13 @@
         }
     },
     "data_import" :{
+        "see_imported_posts" : "See imported posts",
+        "import_another_csv" : "Import another CSV file",
+        "import_complete" : "Your import is complete",
+        "complete_info" : "The data from your CSV spreadsheet, <em>{{filename}}</em>, was successfully imported into your <a href=\"/settings/survey/{{form_id}}\">{{form_name}}</a> survey.",
+        "import_another" : "Import another CSV file",
+        "nearly_complete" : "Your import is nearly complete",
+        "finish_info" : "The data from your CSV spreadsheet is still being processed. You're welcome to leave this page or wait until the process is complete. Either way, we'll notify you when your data is ready.",
```

Comment:
The import will at this stage still complete if they close the browser but they won't receive a notification. The text comes from 
https://github.com/ushahidi/platform-pattern-library/blob/architecture/assets/templates/layouts.settings-import-complete.hbs

What's a better phrasing? Navigate around the application, or navigate to another part of the application? If they stay inside the application they will receive a notification.